docs/src/pages/demos/dividers/SubheaderDividers.js

```diff
@@ -10,22 +9,23 @@ import BeachAccessIcon from '@material-ui/icons/BeachAccess';
 import Divider from '@material-ui/core/Divider';
 import Typography from '@material-ui/core/Typography';
 
-const styles = theme => ({
+const useStyles = makeStyles(theme => ({
   root: {
     width: '100%',
     maxWidth: 360,
     backgroundColor: theme.palette.background.paper,
   },
   dividerFullWidth: {
-    margin: `5px 0 0 ${theme.spacing(2)}px`,
+    margin: theme.spacing(0.625, 0, 0, 2),
```

Comment:
We could potentially support, but we don't yet.
```jsx
margin: theme.spacing('5px', 0, 0, 2),
```

I would rather remove the indirection:

```suggestion
    margin: `5px 0 0 ${theme.spacing(2)}px`,
```